docs/versions/content-patterns.md

```diff
@@ -0,0 +1,313 @@
+   * **Classic**: All features from all solutions are visible by default using the classic, multilayered navigation menus. You can customize which features are visible individually.
+
+3. If you selected the **Classic** solution view, you can customize the **Feature visibility** as you need it to be for that space.
+
+   :::{note} 
+   Even when disabled in this menu, some Management features can remain visible to some users depending on their privileges. Additionally, controlling feature visibility is not a security feature. To secure access to specific features on a per-user basis, you must configure Kibana Security.
+   :::
+4. Customize the avatar of the space to your liking.
+5. Save your new space by selecting **Create space**.
+::::
+
+:::::
+
+You can edit all of the space settings you just defined at any time, except for the URL identifier.
+
+## Sibling bullets 
+
+**Use case:** Requirements, limits, other simple, mirrored facts.
+
+**Number to use:** Ideal is one per option (e.g. one per deployment type). You might consider nested bullets in limited cases.
+
+### Examples
+
+::::{tab-set}
+:group: one-two
+
+:::{tab-item} One
+:sync: one
+
+#### Required permissions
+
+* **Serverless**: `Admin` or equivalent
+* **Stack v9**: `kibana_admin` or equivalent
+
+:::
+:::{tab-item} Two
+:sync: two
+
+#### Create a space
+
+The maximum number of spaces that you can have differs by [what do we call this]: 
+
+* **Serverless**: Maximum of 100 spaces.
+* **Stack v9**: Controlled by the `xpack.spaces.maxSpaces` setting. Default is 1000.
+:::
+::::
+
+## Callouts
+
+**Use case:** Happy differences, clarifications
+
+Some sections don’t apply to, say, Serverless, because we manage the headache for you. Help people to feel like they’re not getting shortchanged with a callout.
+
+If there’s a terminology change or other minor change (especially where x equates with y), consider handling as a note for simplicity.
+
+### Examples
+
+* *In **Manage TLS certificates** section*:
+
+  :::{tip}
+  Elastic Cloud manages TLS certificates for you.
+  :::
+
+* *In a **Spaces** overview*:
+
+  :::{note}
+  In Stack versions 9.0 and lower, **Spaces** are referred to as **Places**.
+
+## Prose (inline)
+
+**Use case:** Clarifying or secondary information
+
+**Number to use:** ~ once per section (use your judgement)
+
+**When to use:** Cases where the information isn’t wildly important, but nice to know, or to add basic terminology change info to overviews
+
+Sometimes, you can just preface a paragraph with version info. 
+
+### Example
+
+If you're managing a Stack v9 deployment, then you can also assign roles and define permissions for a space from the **Permissions** tab of the space settings. 
+
+When a role is assigned to *All Spaces*, you can’t remove its access from the space settings. You must instead edit the role to give it more granular access to individual spaces.
```

Comment:
I don't think this paragraph has any learning in it? If not, let's remove the distraction.

> When a role is assigned to *All Spaces*, you can’t remove its access from the space settings. You must instead edit the role to give it more granular access to individual spaces.